### PR TITLE
Dev 312

### PR DESCRIPTION
**Pull request by lbdyck**
Several updates to address tagging, detection of changes, and null prefix (qualignr=0) issues.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Yes please!
 # Known Issues (maybe solutions...)
 
 ### Git not found (even though you installed it)
-Zigi needs to find the git executable in the 'PATH'. To determine the 'PATH' zigi sources /etc/proifile and .profile.
+Zigi needs to find the git executable in the 'PATH'. To determine the 'PATH' zigi sources /etc/proifile and ~.profile.
```
Comment (MN-Mike) [inline README.md:39]: Did you mean /etc/profile  (and not /etc/proifile) ?

Comment (lbdyck) [reply README.md:39]: There are multiple places to define the path - there is the generic /etc/profile and then there is the users profile found in their home directory which is referenced by the ~.  There is also a bash way that I don't recall :)

Comment (MN-Mike) [reply README.md:39]: I think you have a typo in the /etc/profile,   The text says /etc/proifile (extra i)

Comment (lbdyck) [reply README.md:39]: I believe you are correct - I will fix this.

**Comment by lbdyck** [comment]
Many updates to consider

**Pull request closed without merging**